src/utils/barcode-scan.js

```diff
@@ -1,5 +1,13 @@
 import { BrowserPDF417Reader } from '@zxing/browser';
 
+/*
```

Comment:
JSDoc uses a special `/** */` syntax for comments as oppose to the normal `/* */` syntax of JavaScript multiline comments. That's how JSDoc keep track of their comments as oppose to other ones.

See this with the `/* */` syntax:
<img width="653" alt="Screen Shot 2023-05-24 at 4 25 25" src="https://github.com/codeforpdx/PASS/assets/14917816/f42d54f1-eae1-43f1-86fd-7b3126e89807">

vs. the `/** */` syntax:
<img width="610" alt="Screen Shot 2023-05-24 at 4 25 45" src="https://github.com/codeforpdx/PASS/assets/14917816/f0bcd9b1-840a-4762-a557-129ac7aa115d">

I think you've been doing JavaScript multiline comments instead of JSDoc comments.